content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -314,12 +317,12 @@ Doing so reveals a separate page displaying all the published content related to
 the default :guilabel:`Published` filter from the search bar in the upper-right corner, to reveal
 all the content related to the course - even the non-published content.
 
-Click the :guilabel:`≣ (List View)` icon in the upper-right corner, directly beneath the search bar
-to switch to list view.
+Click the :guilabel:`≣ (bars)` icon in the upper-right corner, directly beneath the search bar to
+switch to list view.
 
-While in :guilabel:`List View`, there is a checkbox on the far left of the screen, above the listed
-courses, to the left of the :guilabel:`Title` column. When that checkbox is clicked, all the course
-contents are selected at once.
+While in list view, there is a checkbox on the far left of the screen, above the listed courses, to
+the left of the :guilabel:`Title` column. When that checkbox is clicked, all the course contents are
+selected at once.
 
 With all the course content selected, double click any of the boxes in the :guilabel:`Is Published`
```

Comment:
I just had to click, not double click. Maybe that's because when testing, all my items were published already? Might want to check this.